nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/RunMongoAggregationIT.java

```diff
@@ -124,6 +124,33 @@ public void testAggregation() throws Exception {
         }
     }
 
+    @Test
+    public void testExtendedJsonSupport() throws Exception {
```

Comment:
You should probably seed the collection with test data that explicitly tests the behavior.

Reply:
Sorry, I don't understood what did you mean. This test queries by date using Extended JSON "$date" notation. There already has a date field in collection seeded by "setup()". What do you suggest exactaly?